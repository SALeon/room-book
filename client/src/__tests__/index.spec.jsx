import React from 'react';
import { shallow } from 'enzyme';

function Foo() {
  return <div>Ddd</div>;
}

it('render SearchField component', () => {
  const component = shallow(<Foo />);
  expect(component).toMatchSnapshot();
});
