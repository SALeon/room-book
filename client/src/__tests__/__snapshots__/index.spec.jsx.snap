// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render SearchField component 1`] = `
<div>
  Ddd
</div>
`;
